src/search/_Search.jsx

```diff
@@ -24,6 +24,12 @@ const propTypes = {
   onItemSelected: PropTypes.func.isRequired,
 };
 
+const ENTER = 13;
```

Comment:
We should use the keycodes pacakge we use in other terra components.

Reply:
Seeing as we already include it in the [dev-site bundle](https://github.com/cerner/terra-dev-site/blob/master/package.json#L100), I'll go ahead and change this

didn't use initially when prototyping this out